lib/bolt/transport/winrm.rb

```diff
@@ -78,30 +75,11 @@ def run_script(target, script, arguments, _options = {})
         with_connection(target) do |conn|
           conn.with_remote_tempdir do |dir|
             remote_path = conn.write_remote_executable(dir, script)
-            if powershell_file?(remote_path)
-              mapped_args = arguments.map do |a|
-                "$invokeArgs.ArgumentList += @'\n#{a}\n'@"
-              end.join("\n")
-              output = conn.execute(<<-PS)
-$invokeArgs = @{
-  ScriptBlock = (Get-Command "#{remote_path}").ScriptBlock
-  ArgumentList = @()
-}
-#{mapped_args}
-
-try
-{
-  Invoke-Command @invokeArgs
-}
-catch
-{
-  Write-Error $_.Exception
-  exit 1
-}
-          PS
+            if Bolt::Transport::Powershell.powershell_file?(remote_path)
```

Comment:
I think include the module so you can just call these methods

Reply:
How about just changing `Bolt::Transport::Powershell.` to `Powershell.`? Seems like this solution cuts down on characters but helps disambiguate methods like `run_script`.